### PR TITLE
Create basic directory with linter

### PR DESCRIPTION
**Pull request by dshelters**
Set up basic directory and set up linter.

### CODE REVIEW

File: _README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
-# Project Name
+# Ribbit
```
Comment (misteryeo) [inline _README.md]: I like our new project name ha!

File: _eslintrc_js.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+module.exports = {
```
Comment (misteryeo) [inline _eslintrc_js.txt]: Update this to JS file, not text file

File: server/_eslintrc_js.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
```
Comment (misteryeo) [inline server/_eslintrc_js.js]: Update the file name to remove duplicate .js

Comment (dshelters) [inline server/_eslintrc_js.js]: Thanks!

Comment (dshelters) [inline server/_eslintrc_js.js]: Done!

**Pull request merged**